server/svix-server/src/v1/utils/mod.rs

```diff
@@ -462,6 +462,32 @@ pub fn get_unix_timestamp() -> u64 {
         .as_secs()
 }
 
+#[derive(Deserialize, JsonSchema)]
+pub struct AppEndpointParams {
```

Comment:
Either "App" and "Endp" or "Application" and "Endpoint".
I think the latter, and I think we should use it consistently in all of them.